yokai/enum-bundle/2.2/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Yokai\\EnumBundle\\YokaiEnumBundle": ["all"]
```

Comment:
Flex is smart enough to install this bundle without a recipe. Just make sure to specify "symfony-bundle" as a type in composer.json

Reply:
I understand that. In the next major version of this repository, it will become less coupled to symfony, and transform to a library.

As for api platform, I was expecting to distribute everything in the same repository : the library, the symfony bridge, ...

I was expecting that i can add a bundle recipe for a package that is not marked as a bundle explicitely. But regarding the automated tests, I believe it is not possible. Do you confirm ?

Reply:
> I was expecting that i can add a bundle recipe for a package that is not marked as a bundle explicitely.

You can, but that is an edge case. 

> But regarding the automated tests, I believe it is not possible. Do you confirm ?

Im not sure what you mean? The review bot will not be happy if you submit a `library`, that is correct.